metadata/metadata.go

```diff
@@ -91,9 +91,9 @@ func (md MD) Copy() MD {
 	return Join(md)
 }
 
-// Join joins any number of MDs into a single MD.
+// Join joins any number of mds into a single md.
```

Comment:
This was probably fine the way it was before.  Or:

Join combines mds into a single MD.